opts/address_pools.go

```diff
@@ -0,0 +1,79 @@
+package opts
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	types "github.com/docker/libnetwork/ipamutils"
+)
+
+// PoolsOpt is a Value type for parsing the default address pools definitions
+type PoolsOpt struct {
```

Comment:
Seems like we really want `type PoolsOpt []*types.NetworkToSplit`, though I'm not sure why that would be a pointer.

Reply:
@selansen could you have a look at this one? (also why we need a pointer)?

Reply:
I had discussion with Brian over phone on this. when we expand this feature to add global scope we will be adding more attribute to this structure and hence keeping struct as it is.